lib/Rex/Commands/Iptables.pm

```diff
@@ -109,12 +127,13 @@ Open a port for inbound connections.
 =cut
 
 sub open_port {
-
-  my ( $port, $option ) = @_;
+  my (@params) = @_;
```

Comment:
No, I just met this code:

    sub _iptables_list {
       my ( %tables, $ret );
       my (@lines) = @_; 

And supposed that this is author's code style. Also fixed.